pages/clients.html

```diff
@@ -6,13 +6,16 @@
 <div class="compare">
 	<form action="/" method="get" id="compare-form" class="compare-form">
 		<div class="settings-title">
-			Select WebViews to compare
+			Select WebViews or upsream browsers to compare
 			<div class="settings-title-actions">
 				<button type="button" id="compare-check-all-button" class="button" data-label-toggle="Uncheck all"
 					data-checked="true">Check all</button>
 				<div class="button-group">
-					<button type="button" id="compare-mobile-button" class="button">Mobile</button>
-					<button type="button" id="compare-desktop-button" class="button">Desktop</button>
+					<button type="button" id="compare-android-button" class="button">Android</button>
+					<button type="button" id="compare-ios-button" class="button">iOS</button>
+					<!-- hidden for now -->
```

Comment:
Should we remove these if we aren't using them?

Reply:
The desktop and mobile buttons might be useful at some point but I wanted to highlight the new ones for now so I hid them. If we remove them here the JS fails therefore CSS display:none.

Reply:
Can we remove the JS as well? We have it in the git history if we ever want to add this back.